app/hotels/src/filter/FilterStripe.js

```diff
@@ -17,26 +18,55 @@ const styles = {
   },
 };
 
-export default class FilterStripe extends React.Component<{||}> {
-  handleChange = () => {
-    // TODO add onChange prop, pass changed values via callback
+type Props = {|
+  onChange: FilterParametersType => void,
+  filter?: FilterParametersType,
+|};
+
+const MIN_PRICE = 0;
+const MAX_PRICE = 300;
+
+export default class FilterStripe extends React.Component<Props> {
+  handleStarsChange = () => {
+    // TODO stars filter handler
+  };
+
+  handlePriceChange = ({
+    minPrice,
+    maxPrice,
+  }: {
+    minPrice: number,
+    maxPrice: number,
+  }) => {
+    const filter = (({}: any): FilterParametersType);
+    if (minPrice !== MIN_PRICE) {
+      filter.minPrice = minPrice;
+    }
+    if (maxPrice !== MAX_PRICE) {
+      filter.maxPrice = maxPrice;
+    }
```

Comment:
I don't think all this is a responsibility of `FilterStripe`. It's just a stripe so it should care only about being stripe (not about prices and stars and user rating and ... and ...). 😊